kroxylicious-app/pom.xml

```diff
@@ -198,6 +202,35 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                    <groupId>io.fabric8</groupId>
+                    <artifactId>docker-maven-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>build</goal>
+                                <goal>save</goal>
```

Comment:
Not strictly necessary and adds a few seconds to the build, but symmetrical with the operator build.